openedx/adg/lms/webinars/models.py

```diff
@@ -331,13 +365,41 @@ def get_webinar_update_recipients_emails(self):
             )
         )
 
+    @classmethod
+    def clone(cls, webinar_to_clone):
+        """
+        Clone the given webinar as draft
+
+        Arguments:
+            webinar_to_clone (Webinar): Webinar that we want to clone
+        """
+        old_co_hosts = webinar_to_clone.co_hosts.all()
+        old_panelists = webinar_to_clone.panelists.all()
+
+        webinar_to_clone.pk = None
+        webinar_to_clone.id = None
+
+        webinar_to_clone.is_published = False
+        webinar_to_clone.is_cancelled = False
+        webinar_to_clone.modified_by = None
+        webinar_to_clone.modified = now()
```

Comment:
Why we are setting date for modified but not the modified_by? We should actually use the request.user and set modified by and created by.

Reply:
It is set automatically by save method. It has already been overridden before to cater for this.